### PR TITLE
STOR-1077: promote CSIInlineVolumeAdmission feature gate to GA

### PR DESCRIPTION
**Pull request by dobsonj**
https://issues.redhat.com/browse/STOR-1077
/hold for https://github.com/openshift/cluster-kube-apiserver-operator/pull/1434 to merge first (required to remove the SCC toggle in the test)
/cc @openshift/storage

### CODE REVIEW

File: test/extended/storage/inline.go [hunk 1]
```diff
@@ -39,17 +36,8 @@ var _ = g.Describe("[sig-storage][Feature:CSIInlineVolumeAdmission][Serial]", fu
...
-				g.Skip("this test is only expected to work with TechPreviewNoUpgrade clusters")
-			} else {
-				// TODO: remove the SCCVolumeToggle when CSI volumes are allowed by the default SCC's (i.e. after TechPreview)
-				g.By("adding restricted-v2 SCC permission to use inline CSI volumes")
-				sccVolumeToggle = NewSCCVolumeToggle(oc, "restricted-v2", securityv1.FSTypeCSI)
-				sccVolumeToggle.Enable()
-			}
 			exutil.PreTestDump()
 
 			// create the secret to share in a new namespace
```
Comment (dobsonj) [inline test/extended/storage/inline.go:43]: TODO: address deads2k's [comment here](https://github.com/openshift/origin/pull/27682#discussion_r1100556366).

**Comment by dobsonj** [comment]
Oh, this PR also depends on the shared resource driver going GA, because the e2e test uses that driver and it's only deployed for TechPreview clusters right now.

**Comment by dobsonj** [comment]
> Oh, this PR also depends on the shared resource driver going GA, because the e2e test uses that driver and it's only deployed for TechPreview clusters right now.

This is addressed by https://github.com/openshift/cluster-storage-operator/pull/335

**Comment by dobsonj** [comment]
> > Oh, this PR also depends on the shared resource driver going GA, because the e2e test uses that driver and it's only deployed for TechPreview clusters right now.
> 
> This is addressed by [openshift/cluster-storage-operator#335](https://github.com/openshift/cluster-storage-operator/pull/335)

That was reverted in https://github.com/openshift/cluster-storage-operator/pull/343 so I've added the test skip back for non-techpreview clusters (where we don't have the shared resource driver). This should avoid the failures in the last run of [e2e-aws-ovn-serial](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/27713/pull-ci-openshift-origin-master-e2e-aws-ovn-serial/1630165561679810560).
I'd still like to remove the SCCVolumeToggle though code since https://github.com/openshift/cluster-kube-apiserver-operator/pull/1434 merged and this test should not need to modify the SCC anymore.

**Comment by dobsonj** [comment]
/unhold
/retitle STOR-1077: promote CSIInlineVolumeAdmission feature gate to GA

**Review by bertinatto — COMMENTED** [review]
/lgtm

**Pull request merged**